### PR TITLE
Clean up PutRelativeFile docs.

### LINKED ISSUE #65: PutRelativeFile doesn't make it clear that the response type should be JSON

### PR DESCRIPTION
**Pull request by tylerbutler**
Use consistent doc formatting for responses, and clarify that the
response is JSON.

Fixes #65.

### CODE REVIEW

File: docs/docs_source/wopi/files/PutRelativeFile.rst [hunk 1]
```diff
@@ -54,34 +54,41 @@ PutRelativeFile
...
+Name
+    The **string** name of the newly created file without a path. **This is a required value in all PutRelativeFile
+    responses.**
 
-    The **string** URI to the :ref:`CheckFileInfo` URI of the newly created file on the host.
-    **This is a required value in all PutRelativeFile responses.**
+Url
+    The :ref:`CheckFileInfo` URI, as a **string**, of the newly created file on the host, including the :term:`file
+    ID` for the new file and a valid :term:`access token`. **This is a required value in all PutRelativeFile
+    responses.**
```
Comment (MarcinJuraszek) [inline docs/docs_source/wopi/files/PutRelativeFile.rst]: this seems unclear to me. What Url? Edit? View? Download?

Comment (tylerbutler) [inline docs/docs_source/wopi/files/PutRelativeFile.rst]: The CheckFileInfo URL. Any suggestions for making it clearer?

Comment (MarcinJuraszek) [inline docs/docs_source/wopi/files/PutRelativeFile.rst]: Isn't it the same thing as a wopi source url? The one you have to substitute in URLs from Discovery? What is it called there?

Comment (tylerbutler) [inline docs/docs_source/wopi/files/PutRelativeFile.rst]: Nope, they're not the same thing:

>The `WOPISrc` parameter tells Office Online the URL of the host’s WOPI Files endpoint. In other words, it is a URL of the form `http://server/<...>/wopi*/files/(id)`, where `id` is the file id of the file.

This URL that needs to be returned here is the CheckFileInfo URL, which includes an access token. The WOPISrc never includes an access token, hence the difference.

**Pull request merged**